Rawl_B502/forshluinti_úa_náir.ttl

```diff
@@ -0,0 +1,462 @@
+@base <http://example.com/Rawl_B502/forshluinti_úa_náir.ttl>.
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix foaf:  <http://xmlns.com/foaf/0.1/> .
+@prefix rel: <http://purl.org/vocab/relationship/>.
+@prefix owl: <http://www.w3.org/2002/07/owl#> .
+@prefix dcterms: <http://purl.org/dc/terms/> .
+@prefix dctype: <http://purl.org/dc/dcmitype/> .
+@prefix irishRel: <http://example.com/earlyIrishRelationship.ttl#> .
+@prefix irishTitle: <http://example.com/earlyIrishTitles.ttl#> .
+@prefix prov: <http://www.w3.org/ns/prov#> .
+
+<>
+    a dctype:Dataset;
+    dcterms:title "Forshluinti Úa Náir"@sga;
+    dcterms:isFormatOf <http://www.ucc.ie/celt/published/G105003/text003.html>;
+    dcterms:format "text/turtle".
+
+<#Cathlaige>
+    a foaf:Person;
+    irishRel:genName "Cathlaige";
+    rdfs:comment: "[Forshluinti Úa Náir] Cenél Cathlaige".
```

Comment:
The file consists entirely of the "forshluinti úa Náir" ("the foremost names of Uí Náir"). That is, it is simply a list of population groups. I added the title in square brackets becuase it would otherwise not be apparent how the population group related to anything. Unless this would be obvious from the filename?